cypress/support/pageBuilder/pbDeleteBlockCategories.js

```diff
@@ -0,0 +1,40 @@
+import { GraphQLClient } from "graphql-request";
+
+Cypress.Commands.add("pbDeleteBlockCategories", () => {
+    cy.pbListBlockCategories().then(categories => {
+        cy.login().then(user => {
+            const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
+                headers: {
+                    authorization: `Bearer ${user.idToken.jwtToken}`
+                }
+            });
+
+            const mutation = `
```

Comment:
Since this string is constant / doesn't change, let's move the whole const above the `Cypress.Commands.add("pbDeleteBlockCategories", () => {` line. We can do this for the rest of the tests as well.